.github/pull_request_template.md

```diff
@@ -7,6 +7,7 @@
 
 <!-- Backend -->
 ### Backend ###
+- [ ] [Codingkonventions](https://it-at-m.github.io/Wahllokalsystem/technik/coding_conventions/) beachtet
```

Comment:
`Codingconventions` oder `Coding Konventionen` - für deutsch oder englisch entscheiden 😉

Reply:
fixed mit [23ef1f3](https://github.com/it-at-m/Wahllokalsystem/pull/636/commits/23ef1f3b93e07b28d59f5e4b9d2f660599c2f903)